CRM/Case/Form/ActivityToCase.php

```diff
@@ -98,7 +98,6 @@ public function buildQuickForm() {
         'extra' => ['contact_id'],
         'params' => [
           'case_id' => ['!=' => $this->_currentCaseId],
-          'case_id.is_deleted' => 0,
```

Comment:
With this removed, then because of the line right above where it sets `case_id`, the `elseif (empty($params['params']['case_id'])) {` in getList doesn't kick in, so this dropdown then includes deleted cases.

jenga

Reply:
OFFS!
Ok, one more try...

Reply:
Cool.
There's now an edge case where `'params' => ['case_id' => ['=' => 1234]]` doesn't work but close enough!